src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -99,6 +100,20 @@ export default class EnhancedReactTable<T> extends React.Component<IEnhancedReac
 
     private shouldSetState:boolean;
 
+    private rawDataSelector = (state:IEnhancedReactTableState, props:IEnhancedReactTableProps<T>) => props.rawData;
+    private columnsSelector = (state:IEnhancedReactTableState, props:IEnhancedReactTableProps<T>) => props.columns;
+    private columnVisibilitySelector = (state:IEnhancedReactTableState) => state.columnVisibility;
+    private visibleColsSelector = createSelector([this.columnsSelector, this.columnVisibilitySelector],
+        (columns:IColumnDefMap, columnVisibility:IColumnVisibilityState) => this.resolveVisible(columns, columnVisibility));
+    private sortableColsSelector = createSelector(this.visibleColsSelector, (visibleCols:IColumnDefMap)=>this.resolveSortable(visibleCols));
```

Comment:
a minor suggestion: for consistency always break the line by parameter (move the lambda to a new line like the line above this one). It helps reducing the amount of code on a single line, and increases readability.